tlstunnel.install

```diff
@@ -0,0 +1,6 @@
+bin: [
+  "_build/src/tlstunnel.native" {"tlstunnel"}
```

Comment:
In case the native compiler is not available on the platform it's better to have:
```
"?_build/src/tlstunnel.byte" {"tlstunnel"}
"?_build/src/tlstunnel.native" {"tlstunnel"}
```